test/letters/utils/helpers.unit.spec.js

```diff
@@ -69,6 +68,10 @@ describe('Letters helpers: ', () => {
         expect(getBenefitOptionText(option, 'unavailable', true)).to.be.undefined;
       }, ['monthlyAwardAmount', 'serviceConnectedPercentage']);
     });
+    it('should include the awardEffectiveDate in the text for monthlyAward', () => {
+      const tree = getBenefitOptionText('monthlyAwardAmount', 20, true);
+      expect(tree.props.children[1].props.children[0]).to.contain('The effective date');
```

Comment:
I got that to work by doing that and then testing `tree.text()`. Thanks for the suggestion! So much better this way.